installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -0,0 +1,92 @@
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms]
+before you attempt to install an {product-title} cluster in such an environment.
+====
+
+.Prerequisites
+
+* Provision
+xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-infrastructure-user-infra.adoc[leveloffset=+1]
+
+include::modules/installation-network-user-infra.adoc[leveloffset=+2]
+
+include::modules/installation-dns-user-infra.adoc[leveloffset=+2]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/cli-installing-cli.adoc[leveloffset=+1]
+
+include::modules/installation-initializing-manual.adoc[leveloffset=+1]
+
+include::modules/installation-bare-metal-config-yaml.adoc[leveloffset=+2]
+
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
+
+include::modules/installation-user-infra-generate-k8s-manifest-ignition.adoc[leveloffset=+1]
+
+[id="creating-machines-ibm-z"]
+== Creating {op-system-first} machines
+
+Before you install a cluster on IBM Z infrastructure that you provision,
+you must create {op-system} machines for it to use. Follow either the steps
+to use an ISO image or network PXE booting to create the machines.
+
+include::modules/installation-user-infra-machines-iso.adoc[leveloffset=+2]
+
+//include::modules/installation-user-infra-machines-pxe.adoc[leveloffset=+2]
+
+include::modules/installation-installing-bare-metal.adoc[leveloffset=+1]
+
+include::modules/cli-logging-in-kubeadmin.adoc[leveloffset=+1]
+
+include::modules/installation-approve-csrs.adoc[leveloffset=+1]
+
+include::modules/installation-operators-config.adoc[leveloffset=+1]
+
+include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
+
+include::modules/registry-configuring-storage-baremetal.adoc[leveloffset=+3]
+
+include::modules/installation-registry-storage-non-production.adoc[leveloffset=+3]
+
+include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
+
+.Next steps
+
+* xref:../../installing/install_config/customizations.adoc#customizations[Customize your cluster].
```

Comment:
Please double-check these next steps.

Reply:
@dbenoit17 Can you please review the Next steps for me? Thank you!

Reply:
I have no additional input for this. Let me see if anyone on the team can help.

Reply:
I think this looks ok - I'll have another read through on Monday - one exception is that the https://docs.openshift.com/container-platform/4.2/installing/install_config/customizations.html includes the "cluster-samples-operator" which is not available yet for Z, so maybe we can just make a note of that - im not sure the best way to do that though.

Reply:
@kalexand-rh Can you please help with adding the note to the right place?

Reply:
@zzoubkova, after the release notes are updated, I suggest adding a note to https://docs.openshift.com/container-platform/4.2/installing/install_config/customizations.html that says something like "If you installed your cluster on IBM Z, not all cluster customizations are available. See the <link to the section of the release notes about the Z restrictions>."

Reply:
@kalexand-rh Good solution! Let's add the note later after we have the link to Release notes.

Reply:
PR's here: https://github.com/openshift/openshift-docs/pull/18523